qabelbox/src/main/java/de/qabel/android/config/AndroidPersistence.java

```diff
@@ -0,0 +1,187 @@
+package de.qabel.android.config;
+
+import android.content.ContentValues;
+import android.database.Cursor;
+import android.database.SQLException;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import de.qabel.core.config.Persistable;
+import de.qabel.core.config.Persistence;
+import de.qabel.core.exceptions.QblInvalidEncryptionKeyException;
+
+public class AndroidPersistence extends Persistence<QblSQLiteParams> {
+
+    private static final String STR_BLOB = "BLOB";
```

Comment:
Hungarian notation. The `STR` is already contained in `String` so we don't need that